online-workflow/src/main/java/eu/itesla_project/online/OnlineWorkflowImpl.java

```diff
@@ -164,8 +169,16 @@ public void start(OnlineWorkflowContext oCtx) throws Exception {
         for (OnlineApplicationListener l :listeners)
             l.onWorkflowUpdate(new StatusSynthesis(id,StatusSynthesis.STATUS_RUNNING));
 
-        Network network = MergeUtil.merge(caseRepository, parameters.getBaseCaseDate(), parameters.getCaseType(), parameters.getCountries(),
-                loadFlowFactory, 0, mergeOptimizerFactory, computationManager, parameters.isMergeOptimized());
+        Network network = null;
+        if (parameters.getCaseFile() != null ) {
+            network = Importers.loadNetwork(parameters.getCaseFile());
+            if (network == null) {
+                throw new RuntimeException("Case '" + parameters.getCaseFile() + "' not found");
+            }
+        } else {
+            network = MergeUtil.merge(caseRepository, parameters.getBaseCaseDate(), parameters.getCaseType(), parameters.getCountries(),
+                    loadFlowFactory, 0, mergeOptimizerFactory, computationManager, parameters.isMergeOptimized());
```

Comment:
the WF could  possibly work with more than one country (... and merge was used, during the iTesla project)